source/common/router/upstream_request.h

```diff
@@ -187,6 +191,9 @@ class UpstreamRequest : public Logger::Loggable<Logger::Id::router>,
   bool cleaned_up_ : 1;
 
   Event::TimerPtr max_stream_duration_timer_;
+  bool has_early_data_;
+  bool use_alt_svc_ : 1;
```

Comment:
Should the other two bools also be :1 fields? Should they come before max_stream_duration_timer_ so they can all be packed together?

Reply:
done